docs/pages/xy-grid.md

```diff
@@ -81,7 +81,7 @@ The structure of XY grid uses `.grid-x`, `.grid-y`, and `.cell` as its base. Wit
 ## Gutters
 
 The defining feature of the XY grid is the ability to use margin AND padding grids in harmony.
-To define a grid type, simply set `.grid-margin-x` or `.grid-padding-x` on the grid.
+To define a grid type, simply set `.grid-margin-x`/`.grid-margin-y` or `.grid-padding-x`/`.grid-margin-y` on the grid.
```

Comment:
You've written `grid-margin-y` twice - the second alteration should be `grid-padding-y`